.env.sample

```diff
@@ -1,16 +1,20 @@
 # ATTENTION: No URL should have a trailing slash
 
-# Mapbox
-MAPBOX_API_KEY="pk.qe2891fu38e189p3uf1p98u3fp1932fp912839fu12p839fup1239uf1p238fu12p398fp19fj328f0.2D_83924dj1-JLM89N_8da"
+# Mapbox Token
+MAPBOX_API_KEY=pk.123.xyz
 
-# Auth 0
-AUTH0_DOMAIN="myauthzeropath.eu.auth0.com"
-AUTH0_CLIENT_ID="IUONiojufw8273r9POMJIUjuofuwpPwe"
-AUTH0_AUDIENCE="https://my-tree-api-url.io"
+# Auth0
+AUTH0_DOMAIN=myauthzeropath.eu.auth0.com
+AUTH0_CLIENT_ID=7479d17d-2212-4a23-a42c-363b898dc618
+AUTH0_AUDIENCE=https://my-tree-api-url.io
 
 # User management
-USER_DATA_API_URL="https://my-user-management-api.io"
+# Uses the auth0 management API to create the possibility
+# to delete users
+# see https://github.com/technologiestiftung/tsb-trees-api-user-management
+USER_DATA_API_URL=https://my-user-management-api.io
 
-# Trees API
-API_ENDPOINT="https://localhost:8000/my-local-postgres-api" # DEV
-# API_ENDPOINT="https://giessdenkiez-de-postgres-api.vercel.app" # PROD
+# giessdenkiez API
+# See https://github.com/technologiestiftung/giessdenkiez-de-postgres-api
+API_ENDPOINT_DEV=https://localhost:8000/my-local-postgres-api
```

Comment:
This should be `API_ENDPOINT` right? We don't use both `DEV` and `PROD` anymode.